src/Microsoft.OData.Edm/Vocabularies/AlternateKeysVocabularyModel.cs

```diff
@@ -0,0 +1,68 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="AlternateKeysVocabularyModel.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.IO;
+using System.Reflection;
+using System.Xml;
+using Microsoft.OData.Edm.Csdl;
+using Microsoft.OData.Edm.Validation;
+
+namespace Microsoft.OData.Edm.Vocabularies.Community.V1
+{
+    /// <summary>
+    /// Representing Core Vocabulary Model.
+    /// </summary>
+    public static class AlternateKeysVocabularyModel

```

Comment:
I can't make sure the name "AlternateKeysVocabulary*" is ok or not. 

It seems "AlternateKeys is only one part of Community Vocabulary" although there is only alternate keys in the community vocabulary currently.